Pipfile

```diff
@@ -6,22 +6,16 @@ verify_ssl = true
 [packages]
 
 [dev-packages]
-apache-airflow = "==1.10.15"
-apache-airflow-backport-providers-amazon = "*"
-apache-airflow-backport-providers-ftp = "*"
-apache-airflow-backport-providers-http = "*"
-apache-airflow-backport-providers-sftp = "*"
-apache-airflow-backport-providers-slack = "*"
-apache-airflow-backport-providers-ssh = "*"
+apache-airflow = "==2.3.0"
+apache-airflow-providers-amazon = "*"
+apache-airflow-providers-ftp = "*"
+apache-airflow-providers-http = "*"
+apache-airflow-providers-sftp = "*"
+apache-airflow-providers-slack = "*"
+apache-airflow-providers-ssh = "*"
 boto3 = "*"
-cryptography = "==3.4.8"
-flask-openid = "==1.3.0"
-flask-sqlalchemy = "==2.4.4"
-jinja2 = "==2.11.3"
-lxml = "==4.7.1"
-markupsafe= "==2.0.1"
-marshmallow = "==2.21.0"
-moto = "==2.0.6"
+importlib_metadata = "*"
```

Comment:
suggestion: Take a look at the python version that was used to create the venv.  `importlib_metada` should be added automatically by the build step by the markdown module which is a airflow dependency.  If it's not being added automatically it's likely that you are running into the same issue that I was with cob_datapipeline where for some reason my local pyenv was using my homebrew python instead of my pyenv python set at 3.9.10.